modules/install-osp-deploy-dualstack.adoc

```diff
@@ -56,30 +55,29 @@ networking:
     hostPrefix: 23
   - cidr: fd01::/48
     hostPrefix: 64
-  serviceNetwork: <2>
+  serviceNetwork: <1>
   - 172.30.0.0/16
   - fd02::/112
 platform:
   openstack:
-    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <3>
-    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <4>
-    controlPlanePort: <5>
-      fixedIPs: <6>
-      - subnet: <7>
+    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <2>
+    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <3>
+    controlPlanePort: <4>
+      fixedIPs: <5>
+      - subnet: <6>
           name: subnet-v4
           id: subnet-v4-id
-      - subnet: <7>
+      - subnet: <6>
           name: subnet-v6
           id: subnet-v6-id
-      network: <7>
+      network: <6>
         name: dualstack
         id: network-id
 ----
 
-<1> Dual-stack clusters are supported only with the `TechPreviewNoUpgrade` value.
-<2> You must specify an IP address range in the `cidr` field for both IPv4 and IPv6 address families.
-<3> Specify the virtual IP (VIP) address endpoints for the Ingress VIP services to provide an interface to the cluster.
-<4> Specify the virtual IP (VIP) address endpoints for the API VIP services to provide an interface to the cluster.
-<5> Specify the dual-stack network details that are used by all the nodes across the cluster.
-<6> The CIDR of any subnet specified in this field must match the CIDRs listed on `networks.machineNetwork`.
-<7> You can specify a value for either `name` or `id`, or both.
\ No newline at end of file
+<1> You must specify an IP address range in the `cidr` field for both IPv4 and IPv6 address families.
+<2> Specify the virtual IP (VIP) address endpoints for the Ingress VIP services to provide an interface to the cluster.
+<3> Specify the virtual IP (VIP) address endpoints for the API VIP services to provide an interface to the cluster.
+<4> Specify the dual-stack network details that are used by all the nodes across the cluster.
+<5> The CIDR of any subnet specified in this field must match the CIDRs listed on `networks.machineNetwork`.
```

Comment:
Shall we use the full name of `CIDR` here? I see there is no full name of `CIDR` in the whole page, but I think it is better to have it at least once.

Reply:
I personally don't think it's needed to specify the full name. CIDR is well known and correspond to the name of the field in the openstack resource, so it is easier for the user to identify. For example:
```
$ openstack subnet show capo |grep cidr
| cidr                 | 172.30.0.0/16                        |
```